src/sql/workbench/contrib/connection/browser/connection.contribution.ts

```diff
@@ -176,6 +176,18 @@ configurationRegistry.registerConfiguration({
 			'default': 25,
 			'description': localize('sql.maxRecentConnectionsDescription', "The maximum number of recently used connections to store in the connection list.")
 		},
+		'sql.defaultAuthenticationType': {
+			'type': 'string',
+			'enum': ['SqlAuth', 'AzureMFA', `AzureMFAAndUser`, 'Integrated'],
```

Comment:
Auth type for sql appears to be `SqlLogin` instead of `SqlAuth`

Reply:
I saw that too but since the message said SQL Login in the description I didn't think the user would find this confusing.  I can see if it's easy to change.

Reply:
I guess my concern is not about users being confused. The auth type we get from config service will eventually be saved into the constructed ConnectionProfile, which means evaluation like this `this.authenticationType === ProviderConnectionInfo.SqlAuthentication` could become evaluating `'SqlAuth' === 'SqlLogin'`. And there are a lot of 'SqlLogin' literal checks like this.

https://github.com/microsoft/azuredatastudio/blob/9583fcd56a114b6e40b70c74ade0638dd87908e5/src/sql/platform/connection/common/providerConnectionInfo.ts#L203